tools/flowc/manipulation/xliff.flow

```diff
@@ -21,12 +21,25 @@ prepareXLiffData(program : FiProgram) -> [string] {
 	set2array(res)
 }
 
-saveXliffData(program, filename) {
+saveXliffData(program : FiProgram, filename : string, xliffFilter : string) -> void {
 	if (program.config.verbose >= 1) {
 		fcPrintln("Extracting XLiff data...", program.config.threadId);
 	} 
 	setTranslationSimpleStringsApi();
-	data0 = prepareXLiffData(program);
+
+	program0 = if (xliffFilter != "") {
+		filterModulePrefixes = strSplit2WithoutLeave(xliffFilter, [":"]);
```

Comment:
Normally we use , for seperation but maybe there is a good reason to use :?

Reply:
No good reason, just looked at the separator in bash. I'll change it to ","